lib/modules/manager/npm/post-update/index.ts

```diff
@@ -141,10 +141,7 @@ export async function writeExistingFiles(
     const npmLock = packageFile.managerData.npmLock;
     if (npmLock) {
       const npmLockPath = npmLock;
-      if (
-        process.env.RENOVATE_REUSE_PACKAGE_LOCK === 'false' ||
-        config.reuseLockFiles === false
-      ) {
+      if (process.env.RENOVATE_REUSE_PACKAGE_LOCK === 'false') {
```

Comment:
@rarkins i think we should remove this

Reply:
I'm not sure what the else path is doing. It's somehow manipulating the lockfile (widens some deps)